### ISSUE TITLE
bug: Screen turning black, after opening & closing the notification center on iOS

### ISSUE THREAD

**Issue by Wilson2142** [capacitor-community/privacy-screen#106]
**Plugin version:**
4.2.4 - 5.2.0

**Platform(s):**
iOS 17.5.1 , iphone 13 & iphone 15 pro

**Current behavior:**
When the notification center is opened and then closed, the screen turns black, rendering the app unusable.

**Expected behavior:**
The app should remain usable after opening and closing the notification center.

**Steps to reproduce:**

1. Open the app.
2. Swipe down from the  top-left side of the screen to fully open the notification center.
3. Swipe up from the bottom to close the notification center.

**Other information:**
The issue does not occur on version 4.2.2, but using that version causes the built-in share overlay to break the app.

**Error message**
```
Terminating app due to uncaught exception 'NSInvalidArgumentException', 
reason: 'Application tried to present modal view controller on itself. 
Presenting controller is <UIViewController: 0x103913070>.'
```

**Capacitor doctor:**

```
💊   Capacitor Doctor  💊 

Latest Dependencies:

  @capacitor/cli: 6.1.2
  @capacitor/core: 6.1.2
  @capacitor/android: 6.1.2
  @capacitor/ios: 6.1.2

Installed Dependencies:

  @capacitor/cli: 6.1.2
  @capacitor/core: 6.1.2
  @capacitor/android: 6.1.2
  @capacitor/ios: 6.1.2

[success] iOS looking great! 👌
[success] Android looking great! 👌

```

**Comment by Wilson2142** [comment]
Yes sir, looks like a duplicate to me as well. I'm still experiencing this issue with the latest (5.2.0) version tho.

**Comment by robingenz** [comment]
Closed by #107
